internal/run/test_runner.go

```diff
@@ -115,7 +116,6 @@ func (r *Run) Do(s *scenarios.Scenarios) *RunResult {
 	metricsTick.Close()
 	r.gatherMetrics()
 
-	r.activeScenario.Teardown()
 	r.pushMetrics()
 	fmt.Println(r.result.Teardown())
```

Comment:
Think these also need adding to the teardown, so we print teardown complete after the teardown finishes.

Reply:
I've noticed that there's a few issues around reporting the setup / teardown phase, where for example failed setups have a green checkmark on the report and such stuff. So I'm looking into that as well.